DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingUsingFunctions/metadata.json

```diff
@@ -0,0 +1,98 @@
+{
+    "Slug": "projecting-using-functions",
+    "SourceFileName": "projectingUsingFunctions.java",
+    "Title": "Projecting Using Functions",
+    "DescriptionLines": [
+        "The projected query fields can be further processed by using **JavaScript functions.**",
+        "",
+        "These functions are executed on the server side.<br>",
+        "Be aware their execution time does Not time out the query."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Projection with declared function - query on collection",
+            "Type": "link",
+            "Url": "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-project-query-results#example-iv---projection-with-"
+        },
+        {
+            "Title": "Projecting With Object Literals",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#projecting-with-object-literals"
+        }
```

Comment:
suggesting to add a link to the 'java' Query Overview Demo
(and same for the nodejs)